src/sentry/release_health/metrics.py

```diff
@@ -434,16 +447,19 @@ def run_sessions_query(
         query: QueryDefinition,
         span_op: str,
     ) -> SessionsQueryResult:
-
         return run_sessions_query(org_id, query, span_op)
 
+    # TODO RaduW 23.Jan.2023 Reimplement when the Metrics Layer adds support for timestamps
```

Comment:
Do you want to create a ticket for implementing them? You would need them in the `select` I suppose.

Reply:
Yep, I will create a story first to "Add support for returning timestamps in Metric Layer" and then a ticket to finish conversion of release health to metrics layer.